### DISCUSSION TITLE
New Cooking System

### DISCUSSION THREAD

**Discussion by Dionis404** [sunflower-land/sunflower-land#4605] [Ideas]
**Idea for a New Cooking System in Sunflower Land**

Now that the chicken coops have changed their mechanics and a new crafting system has emerged, I want to propose a new cooking system. Unfortunately, the previous discussion on this topic was deleted due to a block on GitHub. This idea can be fully realized within one of the upcoming seasons.

## Key Changes:

- One kitchen building
- Recipe discovery
- Fuel requirements for cooking certain dishes
- New category - Spices
- Food quality
- Cooking Queue

---

### 1. One Kitchen Building

The idea is that initially, the player builds a campfire, which can then be upgraded to a multifunctional kitchen with various cooking stations. The player will be able to enter the kitchen and place different items there, such as ovens, cutting tables, mixers for dough, and more.

#### Advantages:
- Centralization of cooking processes in one place, improving interaction with the game.
- Variety of cooking stations that can add unique mechanics (for example, different cooking methods such as baking, frying, boiling).
- The ability to place NFT objects that can provide cooking bonuses or serve as kitchen decoration.

#### Possible Issues:
- The need for balancing between the number of available stations and their impact on the gameplay.
- Possible difficulties in inventory management with a large number of items.

---

### 2. Recipe Discovery

The recipe discovery system will allow players to experiment with various ingredients and search for unique combinations for cooking. Players will be able to find recipes in the world by participating in quests or exploring new locations.

#### Advantages:
- Increased quantity and variety of food in the game, fostering greater interest in crafting.
- Creation of a community of players who can share recipes and tips.

#### Possible Issues:
- The necessity to develop a hint system so players can easily find recipes.
- Balancing unique recipes so that they are not too powerful compared to regular ones.

---

### 3. Cooking Fuel

For dishes requiring thermal processing, it will be necessary to use fuel. Initially, players will use wood, then coal (a new resource), and ultimately—oil.

#### Advantages:
- The addition of a new resource creates an extra layer of strategy and resource management.
- Dishes requiring more complex fuel will yield more experience and be more expensive for delivery.

#### Possible Issues:
- The need to balance the availability of fuel to avoid situations where players get stuck due to a lack of resources.
- The introduction of new elements requires a well-designed interface for resource management.

---

### 4. Spices

Spices will be an integral part of cooking in the game. They will enhance the quality of prepared dishes and affect their properties. Players will be able to find or grow spices in their gardens.

#### Advantages:
- Spices add depth and variety to the cooking process.
- Players will be able to experiment with different combinations of spices to create unique dishes.

#### Possible Issues:
- Potential confusion regarding which spices go with which dishes, requiring clear hints in the game.
- Balancing the properties of spices so that they are useful but not overly powerful.

---

### 5. Food Quality

Food quality will affect the amount of experience gained from consuming it, as well as its delivery price. High-quality food will provide more bonuses and attract more customers in game quests and events.

#### Advantages:
- Motivation for players to cook better and experiment with recipes and spices.
- Creation of an economy around cooking, where players can earn more by selling quality food.

#### Possible Issues:
- Balancing the quality system to avoid situations where high-quality food becomes the only option for players.
- The necessity for clear criteria for quality so players understand how to improve their dishes.

---

### 6. Cooking Queue
It is proposed to add a "Cooking Queue" feature to the kitchen, which would become available as the kitchen level increases. This would allow players to automatically queue multiple recipes to cook consecutively, similar to the Planting Queue in the crop machine. The queue could be shared for the entire kitchen or set individually for each cooking station, providing more flexibility in managing the process.

#### Advantages:
- Time Efficiency: Players can set up multiple recipes in advance, reducing the need to monitor the game constantly and enabling better time management.
- Enhanced Gameplay Flow: The cooking process becomes smoother and more engaging, allowing players to focus on other tasks within the game.
- Customization Options: Individual queues for each station would offer players greater control over resource allocation and cooking strategy.

#### Possible Issues:
- Reduced Active Engagement: Automating the cooking process may lower the need for players to actively engage with the kitchen, possibly reducing gameplay interaction.
- Increased Complexity: Managing separate queues for each station could be overwhelming for newer players, particularly if the interface is complex.
- Balance Concerns: Automating cooking might disrupt the game's balance, potentially making it easier to level up without active effort.

---

### Conclusion

This new cooking system will add many interesting mechanics and improve the overall gameplay in Sunflower Land. With it, players will not only be able to enhance their cooking skills but also interact with each other, creating a community of cooks within the game. I would be happy to hear your thoughts or suggestions for improving this idea!

**Comment by ShinKan42** [comment]
A huge system and quite interesting indeed.
Though given the work it will require seems like a looooooooooong-term process to build up.
Like the idea.

**Reply by ShinKan42** [reply]
But btw selling food is against the all-along principle of not straight buying XP

**Reply by Dionis404** [reply]
> But btw selling food is against the all-along principle of not straight buying XP

By "sales," I meant deliveries.

**Comment by Dionis404** [comment]
I added the idea for automated cooking.
